### PR TITLE
allowed for parsing using names

### PR DESCRIPTION
**Pull request by pariterre**

### CODE REVIEW

**Comment by pariterre** [comment]
Don't know if it ever be useful, but now we can slice FrameDependentArrays using labels like so:
```python
import pyomeca

c = pyomeca.Markers3d.from_c3d("my_beloved_c3d.c3d")
print(c[:, 1:2, :])  # print markers 1 and 2 using slice
print(c[:, [1, 2], :])  # print markers 1 and 2 using list
print(c[:, "Marker1", :])  # print marker labelled Marker1
print(c[:, ["Marker1"], :])  # print marker labelled Marker1 using list
print(c[:, [1, "Marker3", 2, "Marker4"], :])  # print markers 1, Marker3, 2 and Marker4 in that order

c[:, 1, :] = 0  # Modify the marker 1 using direct assignment
c[:, [2, 3], :] = 6  # Modify the markers 2 and 3 using list
c[:, "Marker1", :] = 1  # Modify the marker Marker1 using direct assignment
c[:, [4, "Marker2", 5], :] = 2  # Modify the markers 2, Marker2 and 3 using list
print(c[:, [1, 2, 4, "Marker1"], 0])  # print first frame of some modified things

```

**Comment by pariterre** [comment]
I also added the capability to get all the time frames from ```c['my_favorite_label']```
You can refer to the tests to see all the way to get/set data.
@romainmartinez , you can review!

File: tests/test_fileio.py [hunk 1]
```diff
@@ -108,3 +109,91 @@ def test_analogs(idx, names, expected_shape, expected_values, extension):
...
+def test_slicing_data():
+    c = Analogs3d.from_c3d(MARKERS_ANALOGS_C3D)
+    n_frames = c.shape[2]
+    analog1 = 'Delt_ant.EMG1'
+    analog2 = 'Delt_med.EMG2'
+    analog1_idx = c.get_index(analog1)[0]
+    analog2_idx = c.get_index(analog2)[0]
+
+    # Accessing all values
+    a = copy.copy(c)
```
Comment (romainmartinez) [inline tests/test_fileio.py]: why not `a.copy()`? I think we do no need to import copy

**Comment by romainmartinez** [comment]
Looks very good to me! You can address the minors comments and then merge :tada:

File: tests/test_fileio.py (hunk 1 above)
Comment (pariterre) [reply tests/test_fileio.py]: Wasn't aware of this method!

**Comment by pariterre** [comment]
@romainmartinez 
Changing stuff discarded your approval...

**Pull request merged**